web/beacon-app/src/features/topics/routes/TopicDetailPage.tsx

```diff
@@ -47,16 +48,16 @@ const TopicDetailPage = () => {
       {isFetchingTopic && <Loader />}
       {topic && wasTopicFetched && (
         <>
-          <TopicQuickView topicID={topicID} />
-          <div className="flex items-center justify-between rounded-md bg-[#F7F9FB] px-6 py-3">
-            <Heading as="h1" className="flex items-center text-2xl font-semibold">
+          {/* <TopicQuickView topicID={topicID} /> */}
+          <div className="flex items-center justify-between rounded-md bg-[#F7F9FB] px-6 py-4">
+            <Heading as="h1" className="flex items-center gap-3 text-2xl font-semibold">
+              <Trans>Topic:</Trans>
               <span className="mr-2" data-cy="topic-name">
                 {topic?.topic_name}
+                &nbsp;
```

Comment:
Adding `gap-3` added a little too much space between the topic name and the tooltip. Placing them within the same element and adding a space resolved the issue.